src/test/java/com/bobocode/dialect/PersistenceUnitPropertiesTest.java

```diff
@@ -0,0 +1,58 @@
+package com.bobocode.dialect;
+
+import com.bobocode.bibernate.configuration.PersistenceUnitProperties;
+import com.bobocode.parser.YamlPropertyParser;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PersistenceUnitPropertiesTest {
+    private static final String H_2 = "H2";
+    public static final String JDBC_URL = "jdbc:h2:mem:testdb";
+    public static final String SA = "sa";
+    public static final String PASSWORD = "password";
+
+    private static Stream<Arguments> invalidProperties() {
+        var parser = new YamlPropertyParser();
```

Comment:
We cannot call this test "Unit" if it depend on the logic of another class (here "YamlParser"). But I think we can leave it as it is, since we decided not to spend a lot of time on unit tests and put more effort on integration tests

Reply:
I guess Unit in the name doesn't refer to Unit Tests but rather to Persistence Unit

Reply:
I think the same, better to leave it :)